assets/js/app/LocusZoom.js

```diff
@@ -59,7 +61,7 @@ LocusZoom.populateAll = function(selector, datasource, layout, state) {
 
 // Convert an integer position to a string (e.g. 23423456 => "23.42" (Mb))
 LocusZoom.positionIntToString = function(p){
-    var places = Math.max(6 - Math.floor((Math.log(p) / Math.LN10).toFixed(9)), 2);
+    var places = Math.min(Math.max(6 - Math.floor((Math.log(p) / Math.LN10).toFixed(9)), 2), 12);
```

Comment:
It looks like this change originated in [commit c97fdc9](https://github.com/statgen/locuszoom/commit/c97fdc995a1aaa5faa79396c5f316120e7a1db8f#diff-8c4275047bc760f446e4203cd9f5e5fdR64). I'm trying to figure out why I made this change, especially considering no associated tests were changed with it. My best guess would be that this function is not doing any type checking or integer casting on the input value... it'll just pump whatever it gets through the math one-liner. One could produce a number greater than 12 by supplying a small enough float value (e.g. 0.0000009 or smaller) and with the new logic increasingly small floats max out at 12 places (while integers behave exactly the same).

That said, I still don't know why I cared about that at the time. Being safer at handling float values (though the function isn't intended to take those) I'd be more inclined to keep this change, but it is more or less unnecessary and a more "correct" change would probably involve some integer casting on the input.